vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -121,6 +121,16 @@ testPackagesInstalled() {
         echo $test "[INFO] Directory ${extractedPackageDir} exists"
         continue
       fi
+
+      # if the downloadLocation is /usr/local/bin verify that the package is installed
+      if [ "$downloadLocation" == "/usr/local/bin" ]; then
+          if command -v "$name" >/dev/null 2>&1; then
+              echo "$name is installed."
+              continue
+          else
```

Comment:
If the command doesn't exist, is it supposed to continue the next check or to exit with error?
The current code is just logging, right?

Reply:
The else calls error so if it doesn't exist, I think it will fail. But atm this only runs if the download location is "/usr/local/bin"

Reply:
the `err` is defined above as 
```err() {
  echo "$1:Error: $2" >>/dev/stderr
}
```
So it doesn't exit but just writing to stderr. If you want to exit, you will need to call exit explicitly, like ln 41.